web/views.py

```
@@ -7401,6 +7418,239 @@ def delete_post(request, post_id):
+def ai_chat(request):
+    """Render the AI chat interface"""
+    return render(request, "web/ai_chat/chat.html")
+
+@login_required
+@require_http_methods(["POST"])
+def create_chat_session(request):
+    """Create a new chat session"""
+    try:
+        session = AIChatSession.objects.create(user=request.user)
+        return JsonResponse({"session_id": str(session.id)})
+    except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)